docs/quickstart.md

```diff
@@ -43,23 +43,12 @@ sudo nexd https://try.nexodus.io
 
 Download the latest release package for your OS and architecture. Each release includes a `nexd` binary and a `nexctl` binary.
 
-*Download `nexd`*
-
-- [Linux x86-64](https://nexodus-io.s3.amazonaws.com/linux-amd64/nexd)
-- [Linux arm64](https://nexodus-io.s3.amazonaws.com/linux-arm64/nexd)
-- [Linux arm](https://nexodus-io.s3.amazonaws.com/linux-arm/nexd)
-- [Mac x86-64](https://nexodus-io.s3.amazonaws.com/darwin-amd64/nexd)
-- [Mac arm64 (M1, M2)](https://nexodus-io.s3.amazonaws.com/darwin-arm64/nexd)
-- [Windows x86-64](https://nexodus-io.s3.amazonaws.com/windows-amd64/nexd.exe)
-
-*Download `nexctl`*
-
-- [Linux x86-64](https://nexodus-io.s3.amazonaws.com/linux-amd64/nexctl)
-- [Linux arm64](https://nexodus-io.s3.amazonaws.com/linux-arm64/nexctl)
-- [Linux arm](https://nexodus-io.s3.amazonaws.com/linux-arm/nexctl)
-- [Mac x86-64](https://nexodus-io.s3.amazonaws.com/darwin-amd64/nexctl)
-- [Mac arm64 (M1, M2)](https://nexodus-io.s3.amazonaws.com/darwin-arm64/nexctl)
-- [Windows x86-64](https://nexodus-io.s3.amazonaws.com/windows-amd64/nexctl.exe)
+- [Linux x86-64](https://nexodus-io.s3.amazonaws.com/qa/nexodus-linux-amd64.tar.gz)
+- [Linux arm64](https://nexodus-io.s3.amazonaws.com/qa/nexodus-linux-arm64.tar.gz)
+- [Linux arm](https://nexodus-io.s3.amazonaws.com/qa/nexodus-linux-arm.tar.gz)
+- [Mac x86-64](https://nexodus-io.s3.amazonaws.com/qa/nexodus-darwin-amd64.tar.gz)
+- [Mac arm64 (M1, M2)](https://nexodus-io.s3.amazonaws.com/qa/nexodus-darwin-arm64.tar.gz)
+- [Windows x86-64](https://nexodus-io.s3.amazonaws.com/qa/nexodus-windows-amd64.zip)
```

Comment:
Need to fix `.tar.gz` to `.tgz` or change the Makefile and build.yml over to `.tar.gz`. I'm taking a poll on Discord for preference!

Reply:
May be worth considering adding the extraction command and copying it to a directory in $PATH. I'm on the fence, so not slowing it down, just throwing it out there!

Reply:
Yes, you're right. It's worth fixing this so the instructions are complete. I'll go ahead and push an extra patch to this branch for it.

Reply:
@nerdalert ok, take another look. It has docs now